python_src/src/lamp_py/tableau/hyper.py

```diff
@@ -163,8 +163,10 @@ def create_local_hyper(self) -> int:
         )
 
         # create local HyperFile based on remote parquet file
+        # log_config = "" disables creation of local logfile
         with HyperProcess(
             telemetry=Telemetry.DO_NOT_SEND_USAGE_DATA_TO_TABLEAU,
+            parameters={"log_config": ""},
         ) as hyper:
```

Comment:
this was spitting  out .log files onto the local machine without this parameter getting passed.